doc/tutorial/climada_hazard_Hazard.ipynb

```diff
@@ -752,9 +824,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
    "metadata": {},
-   "outputs": [],
```

Comment:
The readthedocs build fails because at least one output cell is missing. I found this one. Apparently, our doc setup fails if output gets deleted (or, at least, if we delete it in VSCode. The `outputs` section should never be deleted). The remedy is to re-run this cell, or the entire notebook for good measure. Afterward, commit the notebook again.

Reply:
![Screenshot 2024-07-17 at 14 46 13](https://github.com/user-attachments/assets/5321e9cd-a15b-4464-a685-64e076626973)